src/test/apps.test.js

```diff
@@ -1090,3 +1091,69 @@ describe("App Status component", function () {
   });
 
 });
+
+describe("Breadcrumb Component", function () {
+  before(function () {
+    this.renderComponent = (group, app, task) => {
+      var renderer = TestUtils.createRenderer();
+      renderer.render(
+        <BreadcrumbComponent group={group} app={app} task={task} />
+      );
+      return renderer.getRenderOutput();
+    };
+  });
```

Comment:
:beer: 
I remember this was important on some components to cleanup all handlers and triggering componentWillUnmount, etc.
I think it is always a good idea to do this.